### ISSUE TITLE
trailing slash missing

### ISSUE THREAD

**Issue by schmitch** [Azure/autorest#489]
Hello, If I have a url like: "/api/v1/template/{key}/" it will fail to add the trailing slash to the url:

```
var url = new Uri(new Uri(baseUrl + (baseUrl.EndsWith("/") ? "" : "/")), "api/v1/template/{key}").ToString();
```

**Comment by devigned** [comment]
Good catch. We'll get some eyes on it.

**Comment by stankovski** [comment]
Fixed in 0.14.0

**Comment by ragnarok111** [comment]
Is this fixed? I'm using 0.14.0 and I still have to manually edit the generated code to add trailing slashes.
